OVERVIEW.md

```diff
@@ -0,0 +1,72 @@
+# dPanc
+
+## Description
+dPanc aims to leverage Ethereum network's distributed and immutable nature to provide a platform for diabetics to monitor their health.
+
+## Today's Problem
+Today in the US, a diabetic patient will typically visit their doctor for a checkup once every few months.
+
+The standard checkup involves testing and recording the patient's blood glucose level, and comparing the results with data from previous visits.
+
+If the patient can afford to own a blood glucose testing kit and has the ability to export/record the data from the device, then the doctor can assess the patient's situation better as there are more data points to build a better model of the patient. If not, then the doctor has to rely on the data generated every visit.
+
+A person's blood glucose level can be affected by many different factors such as diet, exercise, and sleep. There is a general "acceptable" range for blood glucose level, but adjusting the level depends on a person-to-person basis so most treatment options rely on self-care.
+
+Unfortunately, the US healthcare system is incredibly expensive for patients so many type 1 diabetics will sacrifice their health to save money.
```

Comment:
Need to capitalize the T in type and D in diabetes and this will require evidence that the health care system is particularly expensive for diabetics